config/config.go

```diff
@@ -95,7 +95,8 @@ const (
 )
 
 type Experimental struct {
-	EnableStargz bool `toml:"enable_stargz"`
+	EnableStargz    bool `toml:"enable_stargz"`
+	EnableReferrers bool `toml:"enable_referrers"`
```

Comment:
Can we hide the details that we use OCI referer API? How about renaming it to `oci_subject`, which means oci image manifest has a subject field associated with nydus image

Reply:
Updated to `enable_referrer_detect`.